daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Anf.scala

```diff
@@ -271,12 +262,8 @@ private[lf] object Anf {
     *  Note: this wrapping is the reason why we need a "second" CPS transform to
     *  achieve constant stack through trampoline.
     */
-  private[this] def transformExp[A](
-      depth: DepthA,
-      env: Env,
-      exp: SExpr,
-      transform: Tx[SExpr, A],
-      k: K[AExpr, A]): Trampoline[A] =
+  private[this] def transformExp[A](depth: DepthA, env: Env, exp: SExpr, k: K[AExpr, A])(
```

Comment:
Might wanna split `k` out too.

```suggestion
  private[this] def transformExp[A](depth: DepthA, env: Env, exp: SExpr)(k: K[AExpr, A])(
```